workflow/quickstarts/express.mdx

```diff
@@ -213,6 +217,6 @@ curl -X POST <DEPLOYMENT_URL>/workflow \
 
 1. Learn how to protect your workflow endpoint from unauthorized access by [securing your workflow endpoint](/workflow/howto/security).
 
-2. Explore our [Express example](https://github.com/upstash/workflow-js/tree/main/examples/express) for a detailed, end-to-end example and best practices.
+2. Explore our [Astro example](https://github.com/upstash/workflow-js/tree/main/examples/astro) for a detailed, end-to-end example and best practices.
```

Comment:
should remain Express

Reply:
lol, I changed the wrong one. it's still express at astro docs :D